ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,503 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Delegate protocol to handle user interaction with the CardView
+@objc(MSFCardDelegate)
+public protocol CardDelegate {
+    /// Called after the Card or the button inside the Card is tapped
+    /// - Parameter card: The Card that was tapped
+    @objc optional func didTapCard(_ card: CardView)
+}
+
+@objc(MSFCardBackgroundStyle)
+public enum CardBackgroundStyle: Int, CaseIterable {
+    case appColor
+    case neutral
+    case custom
+}
+
+/// Card style can be announcement, xSmallHorizontal, xSmallVertical, small, or medium)
+/// announcement has  a title, a subtitle, and a button that are centrally aligned, and a featured image (the card is not clickable other than the button)
+/// small and medium styles have a title, a subtitle, an icon and a featured image. The whole card is clickable
+/// xSmall styles have a title, an icon and an optional subtitle. The whole card is clickable
+@objc(MSFCardStyle)
+public enum CardStyle: Int, CaseIterable {
+    case xSmallHorizontal
+    case xSmallVertical
+    case small
+    case medium
+    case announcement
+
+    var imageHeight: CGFloat {
+        switch self {
+        case .medium:
+            return 128
+        case .small:
+            return 76
+        case .announcement:
+            return 100
+        case .xSmallHorizontal, .xSmallVertical:
+            return .zero
+        }
+    }
+
+    var iconSize: CGSize {
+        switch self {
+        case .medium, .small:
+            return CGSize(width: 16, height: 16)
+        case .xSmallHorizontal, .xSmallVertical:
+            return CGSize(width: 18, height: 18)
+        case .announcement:
+            return .zero
+        }
+    }
+
+    var textAlignment: NSTextAlignment {
+        switch self {
+        case .announcement:
+            return NSTextAlignment.center
+        default:
+            return NSTextAlignment.left
```

Comment:
Seems like my comment disappeared so will post again:
Should this be neutral instead of left for RTL support?